ldk/javascript/examples/self-test-loop/src/tests/index.ts

```diff
@@ -11,7 +11,8 @@ import {
   filesystem,
 } from '@oliveai/ldk';
 
-import { Cancellable } from '@oliveai/ldk/dist/cancellable'
+import { Cancellable } from '@oliveai/ldk/dist/cancellable';
+import { collapseTextChangeRangesAcrossMultipleVersions } from 'typescript';
```

Comment:
What is this?

Reply:
Removed, I have no idea where this thing came from. Perhaps a merge, not sure.